server/app/models/host_node.rb

```diff
@@ -29,6 +29,8 @@ class Error < StandardError
   field :agent_version, type: String
   field :docker_version, type: String
   field :plugins, type: Hash, default: {}
```

Comment:
The `plugins` field is abandoned, and not getting updated anymore? Should we remove it?